app/invocation/invocation_action_card.tsx

```diff
@@ -225,12 +225,12 @@ export default class InvocationActionCardComponent extends React.Component<Props
    * locate the ExecuteResponse that was returned for this particular
    * invocation.
    */
-  fetchActionResult() {
+  fetchActionResult(): CancelablePromise<void> | null {
     let digestParam = this.props.search.get("actionDigest");
     const digest = parseActionDigest(digestParam ?? "");
     if (!digest) {
       alert_service.error("Missing action digest in URL");
-      return;
+      return null;
```

Comment:
The return type `Promise | null` seems pretty unconventional. Usually, functions are either sync or async, not both.

I think it would be better to skip calling this function if the `actionDigest` param is not set, and revert the changes to this func.